CONTRIBUTING.md

```diff
@@ -34,6 +34,16 @@ If you want to work on a feature or a bug fix which does not have an open issue,
 
 In any case, feel free to ask questions if you do not understand a part of the code, or if you are unsure how a particular feature should be implemented.
 
+### Code Formatting
+
+Please use [`uncrustify`](https://github.com/uncrustify/uncrustify) to automatically format the code before committing:
```

Comment:
We might want to say somewhere that the version of uncrustify should be >= 0.7.1.

Reply:
alright I added it in parentheses